src/main/java/api/apiwrappers/OrderApiWrapper.java

```diff
@@ -14,9 +14,32 @@ public Order postOrder(Order order) throws IOException {
     }
 
     public Order getOrder(Order order) throws IOException {
-        String endpoint = baseUrl + "/v2/store/order/" + order.getId().toString();
+        String endpoint = baseUrl + "/v2/store/order/" + order.getId();
         
         return (Order)get(order, endpoint);
     }
 
+    public Order getOrderInventory(Order order) throws IOException {
+        String endpoint = baseUrl + "/v2/store/inventory";
+
+        return (Order)get(order, endpoint);
+    }
+
+    public void deleteOrder(Order order) throws IOException {
+        String endpoint = baseUrl + "/v2/store/order/" + order.getId();
+
+        delete(endpoint);
+    }
+
+    public int getGetOrderResponse(){
```

Comment:
This methods names don't reflect what the methods return. it should be get..ResponseStatus();

Reply:
I've changed the names of the methods to `getGetOrderResponseStatus()`,  `getPostOrderResponseStatus()` and `getDeleteOrderResponseStatus()`.